src/main/java/org/openmrs/addonindex/domain/AddOnInfoAndVersions.java

```diff
@@ -54,6 +54,8 @@
 	private String hostedUrl;
 	
 	private List<AddOnVersion> versions = new ArrayList<>();
+
+	private Integer downloadCounts;
```

Comment:
Need to be clearer what this is, e.g. "downloadCountInLast30Days". (I forget exactly what it is.)